api_tests/identifiers/views/test_identifier_list.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+import urlparse
+from modularodm import Q
+from nose.tools import *  # flake8: noqa
+
+from api.base.settings.defaults import API_BASE
+from website.identifiers.model import Identifier
+
+from tests.base import ApiTestCase
+from tests.factories import (
+    RegistrationFactory,
+    AuthUserFactory,
+    IdentifierFactory
+)
+
+
+class TestIdentifierList(ApiTestCase):
+    def setUp(self):
+        super(TestIdentifierList, self).setUp()
+        self.user = AuthUserFactory()
+        self.user_two = AuthUserFactory()
+
+        self.registration = RegistrationFactory(creator=self.user, is_public=True)
+        self.identifier = IdentifierFactory(referent=self.registration)
+        self.url = '/{}nodes/{}/identifiers/'.format(API_BASE, self.registration._id)
```

Comment:
This should be registrations, not nodes. How does this even work? You shouldn't be able to access a registration through a node sub-route. (Don't research that; it was rhetorical)